src/Packetery/Module/Carrier/OptionsPage.php

```diff
@@ -563,6 +566,7 @@ public function render(): void {
 				'carrierDoesNotSupportCod'               => __( 'This carrier does not support COD payment.', 'packeta' ),
 				'allowedPickupPointTypes'                => __( 'Pickup point types', 'packeta' ),
 				'checkAtLeastTwo'                        => __( 'Check at least two types of pickup points or use a carrier which delivers to the desired pickup point type.', 'packeta' ),
+				'countAvailableVendorsLow'               => __( 'This carrier displays all types of dispensing points at the same time in the checkout (pickup points, Z-boxes).', 'packeta' ),
```

Comment:
"dispensing points" nikde nepoužíváme, je potřeba ověřit, jak to je správně

Reply:
Použiji pickup points, jelikož to používáme tak v doplňku. Nic méně asi správně je pick-up points. Pořeším s @zemekoule bokem.